package.json

```diff
@@ -47,6 +47,7 @@
     "lodash.merge": "^4.3.5",
     "lodash.throttle": "^4.0.1",
     "react-addons-create-fragment": "^15.0.1",
+    "react-addons-shallow-compare": "^15.0.1",
```

Comment:
It isn't needed - I had used that first, then removed it from the template, but not from `packages`